cmd/bosun/conf/notify.go

```diff
@@ -38,33 +38,36 @@ func init() {
 type PreparedNotifications struct {
 	Email  *PreparedEmail
 	HTTP   []*PreparedHttp
-	Print  string
+	Print  bool
+	Name   string
 	Errors []string
 }
 
 func (p *PreparedNotifications) Send(c SystemConfProvider) (errs []error) {
 	if p.Email != nil {
 		if err := p.Email.Send(c); err != nil {
-			slog.Errorf("sending email: %s", err)
+			slog.Errorln("status: error; name: " + p.Name + "; subject: " + p.Email.Subject + "; transport: email; address: " + strings.Join(p.Email.To, ",") + "; body: " + p.Email.Body + "; error: " + err.Error())
```

Comment:
slog.Errorf instead, and then if the fmt strings are the same you can put the in a higher level var or a constant

Reply:
Hi! Thanks for advice. I've also aggregated format for error string generation and moved this format to const